src/main/java/dataInt/Ejercicio3.java

```diff
@@ -0,0 +1,30 @@
+package dataInt;
+
+import com.github.javafaker.Faker;
+
+import java.util.Arrays;
+
+public class Ejercicio3 {
+    public static void main(String[] args) {
+        final var listaNumeros = new int[10];
+        for (int i = 0; i <= listaNumeros.length - 1; i++) {
```

Comment:
final var n = listaNumeros.length;
for(var i = 0; i < n; i++)

Reply:
este for aplica para todos los de abajo

Reply:
Se corrigio!